src/Untagged/TypeCheck.purs

```diff
@@ -56,10 +56,10 @@ instance hasRuntimeTypeRecord ::
   ( RowToList r rl
   , HasRuntimeTypeRecordRL rl
   ) => HasRuntimeType {|r} where
-  hasRuntimeType _ x = hasJsType "object" x && hasRuntimeTypeRecRL (RLProxy :: _ rl) (unsafeToForeign x)
+  hasRuntimeType _ x = hasJsType "object" x && hasRuntimeTypeRecRL (Proxy :: _ rl) (unsafeToForeign x)
 
-class HasRuntimeTypeRecordRL (rl :: RowList) where
-  hasRuntimeTypeRecRL :: RLProxy rl -> Foreign -> Boolean
+class HasRuntimeTypeRecordRL (rl :: RowList Type) where
+  hasRuntimeTypeRecRL :: forall proxy. proxy rl -> Foreign -> Boolean
```

Comment:
Yeah, that's the correct way to do this. Now `RLProxy foo` and `Proxy foo` will work.